mk/utils/glide.mk

```diff
@@ -0,0 +1,28 @@
+# Cross builder helper
+define glide-install
+	glide install -s --update-vendored
+endef
+
+define glide-update
+	glide update -s --update-vendored
```

Comment:
Note, in the near future we'll be removing the need to use `-s` or `--update-vendored`. Just want to share some intent so you know where this is going.